navigation/objects/LICENSE.txt

```diff
@@ -0,0 +1 @@
+This thing was created by Thingiverse user figure, and is licensed under cc.
```

Comment:
Which "thing"? Need to specify which files.

Reply:
This was the file provided by the website along with the model. Are we allowed to edit it?

Reply:
I think so, because we need to specify what "thing" it refers to.